db/models/seeders.js

```diff
@@ -0,0 +1,165 @@
+const { query } = require('../index');
+
+module.exports = {
+  seedStudents: (students) => {
+    return new Promise((resolve, reject) => {
+      students.forEach(async(student, index) => {
+        let q = `
+        INSERT INTO students (id, first_name, last_name, github, cohort_id)
+        VALUES (
+          ${student.id},
+          '${student.firstName}',
+          '${student.lastName}',
+          '${student.github}',
+          ${student.cohortId}
+        )`;
+        try {
+          let insert = await query(q);
+          if (insert.command) {
+            console.log(`Inserted student ${student.firstName}`);
+          }
+          if (index === students.length -1) {
+            try {
+              let insertedStudents = await query(`SELECT * from students ORDER BY id ASC`);
+              resolve(insertedStudents.rows);
+            } catch(error) {
+              console.log(error);
+            }
+          }
+        } catch(error) {
+          reject(error.detail);
+        }
+      });
+    });
+  },
+  seedCohorts: async(cohorts) => {
+    return new Promise((resolve, reject) => {
+      cohorts.forEach(async(cohort, index) => {
+        console.log(cohort.id, cohort.name, cohort.phase);
+        let q = `
+          INSERT INTO cohorts (id, cohort_name, phase)
+          VALUES (${cohort.id}, '${cohort.name}', '${cohort.phase}')
+        `;
+        try {
+          let insert = await query(q);
```

Comment:
Since query is a keyword I think we're stuck here.  Answer may be formalizing `q` as our convention when we're defining query strings means that we don't find ourselves wondering what's up on lines like 44.